dali/kernels/test/tensor_view_test.cc

```diff
@@ -218,5 +218,33 @@ TEST(TensorListView, uniform_list_shape) {
   EXPECT_EQ(infer2, dyn);
 }
 
+namespace {
+template<typename T, size_t N>
+void Verify(const T* data, std::array<long int, N> dims, int idx) {
```

Comment:
Less generic name please - it's case-specific and should be named so.

Reply:
done